screen2.0/src/app/search/page.tsx

```diff
@@ -127,7 +127,7 @@ export default function Search({ searchParams }: { searchParams: { [key: string]
   const [opencCREs, setOpencCREs] = useState<{
     ID: string,
     region: { start: number, end: number, chrom: string },
-    linkedGenes: LinkedGenesData
+    linkedGenes?: LinkedGenesData
```

Comment:
Made this optional for now. Need to change this

Reply:
Gotcha, I will change this